### PR TITLE
Improve track ended event listeners

### PR DESCRIPTION
**Pull request by pnts-se-whereby**
### Description

**Summary:**

**Related Issue:**

### Testing

Test instructions in PWA

### Checklist

-   [x] My code follows the project's coding standards.
-   [x] Prefixed the PR title and commit messages with the service or package name
-   [ ] I have written unit tests (if applicable).
-   [ ] I have updated the documentation (if applicable).
-   [x] By submitting this pull request, I confirm that my contribution is made
        under the terms of the MIT license.

### Additional Information

I left console.logs in the code for reviewer to be able to test it easily.

### CODE REVIEW

File: packages/media/src/webrtc/P2pRtcManager.ts [hunk 1]
```diff
@@ -858,20 +874,24 @@ export default class P2pRtcManager implements RtcManager {
...
-        track.removeEventListener("ended", this._audioTrackOnEnded);
-    }
-    
-    _startMonitoringVideoTrack(track: CustomMediaStreamTrack) {
+    _monitorVideoTrack(track: CustomMediaStreamTrack) {
+        if (this._videoTrackBeingMonitored?.id === track.id) return;
+        
+        this._videoTrackBeingMonitored && console.log("stop monitoring", this._videoTrackBeingMonitored.label);
+        this._videoTrackBeingMonitored?.removeEventListener("ended", this._videoTrackOnEnded);
+        console.log("start monitoring", track.label);
```
Comment (misi-whereby) [inline packages/media/src/webrtc/P2pRtcManager.ts:892]: can we use some logger here or only console?

**Review by misi-whereby — APPROVED** [review]
lgtm

File: packages/media/src/webrtc/P2pRtcManager.ts (hunk 1 above)
Comment (pnts-se-whereby) [reply packages/media/src/webrtc/P2pRtcManager.ts]: It was only to make reviewing easier. Fixed in [a1aa382](https://github.com/whereby/sdk/pull/316/commits/a1aa38231e4745b1fb08dcf4b0c563a09e72dea6)

**Pull request merged**